From 33b8148614c47a091da1a711fb53f94827a9cf51 Mon Sep 17 00:00:00 2001
From: Tim Harder <radhermit@gmail.com>
Date: Sat, 25 Oct 2014 12:30:19 -0400
Subject: [PATCH] Fix Makefile depend file usage

Creating the .depend file requires the config.h file to exist and
suppress make's complaints if the .depend file doesn't exist when
initially parsing the makefile.
---
 Makefile | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git Makefile.orig Makefile
index 9282884..3926145 100644
--- Makefile
+++ Makefile
@@ -14,7 +14,8 @@ OBJ = $(SRC:.c=.o)

 all: sxiv

-$(OBJ): Makefile config.h
+$(OBJ): Makefile
+$(OBJ) .depend: config.h

 depend: .depend

@@ -22,7 +23,7 @@ depend: .depend
 	rm -f ./.depend
 	$(CC) $(CFLAGS) -MM $^ >./.depend

-include .depend
+-include .depend

 .c.o:
 	$(CC) $(CFLAGS) $(CPPFLAGS) -DVERSION=\"$(VERSION)\" -c -o $@ $<
